packages/bpk-component-graphic-promotion/src/BpkGraphicPromo-test.js

```diff
@@ -125,12 +125,4 @@ describe('BpkGraphicPromo', () => {
 
     expect(props.onClick).toHaveBeenCalledTimes(1);
   });
-
```

Comment:
Same with the graphic promo changes, they were merged in after you started the contribution 🙂

Reply:
Nice catch, I have updated this part. Thank you very much!